vertx-mail-client/src/main/java/io/vertx/ext/mail/MailConfig.java

```diff
@@ -371,21 +353,21 @@ public MailConfig setTrustAll(boolean trustAll) {
 
   /**
    * get the NetClientOptions to be used when opening SMTP connections
-   * 
+   *
    * @return the netClientOptions
    */
+  // FIXME - why allow NetClientOptions to be provided?
```

Comment:
when using a custom key store, the NetClientOptions are necessary to the set the correct jks options, see this unit test:
https://github.com/vert-x3/vertx-mail-client/blob/master/src/test/java/io/vertx/ext/mail/MailLocalTest.java#L46